dist/assets/css/main.css

```diff
@@ -1414,6 +1428,19 @@ footer #copyright p a:hover {
           flex-direction: column;
   margin: auto;
 }
+#article main section div div {
+  display: -webkit-flex;
+  display: -ms-flexbox;
+  display: flex;
+  -webkit-flex-direction: row;
```

Comment:
Remove the browser prefixes. These are added automatically by Autoprefixer.
So just
```
display: flex;
flex-direction: row;
```
Will do here, no need to repeat with prefixes.